config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -4655,3 +4655,12 @@
 # DEFAULT: false
 #gradebook.enable.finalgrade=false
 
+# ###############################################################
+# SAK-30461 Add academic and social alerts icons to portal topbar
+#
+# Set this to false to hide the bullhorn icons in the topbar and
+# disable bullhorn event harvesting.
+#
+# true/false Defaults to true (on)
+# ###############################################################
+#useBullhornAlerts=false
```

Comment:
Could we prefix this with `portal.` and most of the portal configuration uses a dot separator rather than camel case, so maybe `portal.use.bullhorn.alert`